docs/Github-workflow.md

```diff
@@ -0,0 +1,69 @@
+The Performance team uses GitHub to manage all code and related discussions for the Performance Lab plugin. Please follow the workflow below to ensure that all issues are properly tracked.
+
+## Opening an issue
```

Comment:
Just an idea: You have below other sections of the same type called "Pull requests" and then "Trac tickets". How about we make the `h2` here a single "Issues" headline and then the "Opening an issue" right below it, which would bring it to the same heading level as "Working on an issue"?

Here's what I mean:
```suggestion
## Issues

### Opening an issue
```

This way the document outline would become:
* GitHub workflow
    * Issues
    * Pull requests
    * Trac tickets
    * POC responsibilities